examples/make-script/makefile

```diff
@@ -9,10 +10,17 @@ OBJ = $(SOURCEC:%.cxx=%.o)
 TARGET ?= $(SOURCEC:%.cxx=%)
 
 # Use the bout-config script to get compiler, flags
-CXX=`bout-config --cxx`
-CFLAGS=`bout-config --cflags`
-LD=`bout-config --ld`
-LDFLAGS=`bout-config --libs`
+CXX:=$(shell bout-config --cxx)
+
+ifneq "$(.SHELLSTATUS)" "0"
+which=$(shell which bout-config)
+$(info Output of which bout-config: $(which))
+$(error Failed to run bout-config.)
```

Comment:
Maybe combine these two lines and put something like "is bout-config in $PATH"? Just to make the error a bit nicer